src/otserv.cpp

```diff
@@ -37,96 +33,67 @@
 #include "script.h"
 #include <fstream>
 
-DatabaseTasks g_databaseTasks;
-Dispatcher g_dispatcher;
-Scheduler g_scheduler;
+extern Scripts* g_scripts; // Holds '*.lua' scripts from '/data'. Extern means it's defined elsewhere.
+
+DatabaseTasks g_databaseTasks;	// Runs on separate thread, handles database queries
+Scheduler g_scheduler;			// Runs on separate thread, handles planned executions
+Dispatcher g_dispatcher;		// Runs on separate thread, handles immediate executions
 
-Game g_game;
-ConfigManager g_config;
-Monsters g_monsters;
-Vocations g_vocations;
-extern Scripts* g_scripts;
-RSA g_RSA;
+Game g_game;			// Holds and mutates game state (map, players, monsters, NPCs, etc)
+ConfigManager g_config; // Holds configuration from '/config.lua'
+Monsters g_monsters;	// Holds monster definitions from /data/monster (XMLs)
+Vocations g_vocations;	// Holds vocations definitions from '/data/XML/vocations.xml'
+RSA g_RSA;				// Holds private key from '/key.pem' and decrypts data
```

Comment:
Can anyone help me figure out what's going on with indentation?
Git is messing them up somehow... 
![image](https://user-images.githubusercontent.com/23069432/67235237-18464c80-f41d-11e9-9f6b-93e3af53db66.png)

Reply:
Maybe it's related to .editorconfig?
Apparently it's configured to use spaces, which can cause this sort of behavior.

Reply:
GitHub uses tabsize 8 instead of 4.
This is how the code actually looks like?
https://github.com/otland/forgottenserver/blob/96153c4565d0ecf6f9537eb7cda5fa886f26e94f/src/otserv.cpp?ts=4

Reply:
@Znote that might be causing indentation to look "random"